File: finance_client/src/app/user/data/types/organisation.ts
export interface Organisation {
  address?: string;
  city?: string;
  country?: string;
  deletedBy?: string;
  deletedFlag?:string;
  deletedTime?: Date;
  email?: string;
  id?: number;
  modifiedBy?: string;
  modifiedFlag?: string;
  modifiedTime?: Date;
  organisationName?: string;
  phone?: string;
  postedFlag?: string;
  postedTime?: Date;
  reason?: string;
  status?: string;
  verifiedBy?: string;
  verifiedFlag?: string;
  verifiedTime?: Date;
}

File: finance_server/src/main/java/co/ke/emtechhouse/eims/OrganisationComponent/Organisation.java
package co.ke.emtechhouse.eims.OrganisationComponent;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.*;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Entity
public class Organisation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false,  updatable = false)
    private Long id;

    @Column()
    private String organisationName;
    @Column()
    private String email;
    private String phone;
    private String address;
    private String city;
    private String country;
    private String status = "pending";
    private String reason = "-";

    private Character postedFlag;
    private Date postedTime;
    private String modifiedBy;
    private Character modifiedFlag;
    private Date modifiedTime;
    private String verifiedBy;
    private Character verifiedFlag;
    private Date verifiedTime;
    private String deletedBy;
    private Character deletedFlag;
    private Date deletedTime;
}